pkg/kubelet/cm/cpumanager/policy_static.go

```diff
@@ -316,6 +329,10 @@ func (p *staticPolicy) updateCPUsToReuse(pod *v1.Pod, container *v1.Container, c
 func (p *staticPolicy) Allocate(s state.State, pod *v1.Pod, container *v1.Container) (rerr error) {
 	numCPUs := p.guaranteedCPUs(pod, container)
 	if numCPUs == 0 {
```

Comment:
If the number of guaranteed CPUs is 0 for any reason, and the pod sets pod level resources, this will result in the event being emitted, which I don't think is what we want.